.github/workflows/unit_tests.yml

```
@@ -0,0 +1,68 @@
+  pull_request:
+    types: [opened]
+
+name: Unit tests + lints
+
+jobs:
+  test:
+    name: Test Suite
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)